Documentation/helm-values.rst

```diff
@@ -2732,8 +2732,16 @@
      - List of ports from service to be automatically redirected to above backend. Any service exposing one of these ports will be automatically redirected. Fine-grained control can be achieved by using the service annotation.
```

Comment:
There's also references to the now deprecated `localRedirectPolicy` value in network/kubernetes/local-redirect-policy.rst that we may want to update.